src/android/io/branch/BranchSDK.java

```diff
@@ -831,7 +826,8 @@ public LogoutStatusListener (CallbackContext callbackContext) {
         public void onLogoutFinished(boolean loggedOut, BranchError error) {
             if (error == null) {
                 Log.d(LCAT, "no error on logout");
-                this._callbackContext.success(Boolean.toString(loggedOut));
+                branchObjectWrappers = new ArrayList<BranchUniversalObjectWrapper>();
+                this._callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.OK, /* send boolean: is logged out */ loggedOut));
```

Comment:
You should be able to send a true boolean instead of a string this way. sending "FALSE" evaluates to true, but sending false doesn't.